style.css

```
@@ -0,0 +1,165 @@
+@charset "UTF-8";
+
+
+
+body{
```

Sempre use um espaço entre o seletor (body) e o colchete!
```suggestion
body {
```

---

Obrigado